stock_quant_relocate/wizard/quant_move_wizard.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields, api
+
+
+class StockQuantMove(models.TransientModel):
+    _inherit = 'stock.quant.move'
+
+    picking_id = fields.Many2one(
+        comodel_name='stock.picking', string='Picking')
+
+    @api.model
+    def default_get(self, fields):
+        res = super(StockQuantMove, self).default_get(fields)
+        if self.env.context.get('picking_id', False):
+            res = self._catch_product_default_location(res)
+        return res
+
+    def _catch_product_default_location(self, res):
+        quant_obj = self.env['stock.quant']
+        res['picking_id'] = self.env.context.get('picking_id')
+        lines = res.get('pack_move_items', False)
+        for line in lines:
+            if line.get('quant', False):
+                quant = quant_obj.browse(line.get('quant'))
+                if quant.product_id.default_location:
+                    line['dest_loc'] = quant.product_id.default_location.id
+                elif quant.product_id.categ_id.default_location:
+                    line['dest_loc'] = (
+                        quant.product_id.categ_id.default_location.id)
+        return res
+
+    @api.one
```

Comment:
El módulo del que heredas esto, tiene un api.one, pero en la función tiene un for por lo que no se si es correcto hacer esto así